liberty-ls/src/main/java/io/openliberty/tools/langserver/model/envVar/ExpansionVariableInstance.java

```diff
@@ -83,6 +83,9 @@ public CompletableFuture<List<CompletionItem>> getCompletions(Position position)
         };
 
         List<CompletionItem> completionList = documentProperties.keySet().stream().map(s -> s.toString()).filter(filter).map(s -> new CompletionItem(s)).collect(Collectors.toList());
+        for (CompletionItem item : completionList) {
+            item.setInsertText(item.getLabel() + "}");
```

Comment:
When completion is triggered I believe it is possible the end `}` is already present. In that case, you would not want to add it here. Can you add this `}` conditionally depending on the context?